### PR TITLE
Document how to install 007 with zef

### LINKED ISSUE #446: Add instructions for how to install 007 with zef
Then people don't have to mess around with `PERL6LIB` at all.

### PR DESCRIPTION
**Pull request by masak**
As usual, please ignore the generated output in `docs/index.html` and focus on the source in `documentation/README.md`. Kthx.

Closes #446.

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -32,10 +45,7 @@ $ git clone https://github.com/masak/007.git
 [...]
 ```
 
-### Setting an environment variable
-
-We're one step away from running our first 007 program. Before that, we need to
-set an environment variable `PERL6LIB`:
+Finally, we need to set an environment variable `PERL6LIB`:
```
Comment (claes-magnus) [inline documentation/README.md]: 'environment variable'
Suggestion: 'path environment variable'

Comment (masak) [reply documentation/README.md]: I think there is too strong an association with the environment variable actually named `PATH` for the term "path environment variable" to be a net improvement in clarity.

Is there prior art somewhere? I can't recall hearing someone write "path environment variable" before. If there's plenty of examples of other texts using that term for a certain kind of environment variable, I might reconsider.

**Comment by masak** [comment]
> - Include how to uninstall

fce8b09ae4290bbc762a88323c92468fb45bffc1 adds how to upgrade or uninstall using `zef`.

> - I think that there might be benefits with mentioning where the environment variable would be placed (always good to know for the user) and, something that relates to the point above, how to remove it. I don't mean you to write a manual for this, but perhaps a mention of a link (or something like that)?

These are a bit dependent on the exact shell and Linux distribution. There's enough diversity there that I don't feel comfortable giving instructions to a user who truly needs them. In particular, on my system the permanent place to add environment variables is in `~/.bashrc`, but that's because I'm using bash as my shell &mdash; others may not.

I'm in favor of throwing in a link for more information, but I'm not aware of a link that's both shell/distribution-agnostic enough to cover everyone's system, yet specific enough to be useful. The [Wikipedia article](https://en.wikipedia.org/wiki/Environment_variable), for example, is far too general.

By the way, this feels a little bit outside the scope of this PR. :wink: I changed the phrasing in a sentence about environment variables in the section about manual install, but that was mostly to accommodate the new section about zef.

**Comment by claes-magnus** [comment]
Sorry! :)

**Comment by masak** [comment]
Hey, don't say sorry. And thanks for reviewing and merging. :heart:

**Pull request merged**